tracing-appender/Cargo.toml

```diff
@@ -2,12 +2,20 @@
 name = "tracing-appender"
 version = "0.1.0"
 authors = ["Zeki Sherif <zekshi@amazon.com>"]
-edition = "2018"
+license = "MIT"
+readme = "README.md"
 repository = "https://github.com/tokio-rs/tracing"
 homepage = "https://tokio.rs"
 description = """
 Provides an `appender` that writes to a file
```

Comment:
I don't think this description is entirely accurate. This crate provides a file appender _and_ utilities for making a writer non-blocking. Can we update the description to something a little more general?

Reply:
updated